src/en/config-gce.md

```diff
@@ -196,6 +196,7 @@ private_key: |
   E2sZPXYMT8TmQnat35f4+h4FT5MYPHpr/OS27kejHLjxpWkbz3Dziqx6upM+fMiY
   ...
 ```
+!!! Note: The JSON file presents the keys as `client_email:`, `client_id:` and `private_key`, with underscores. Juju will not accept these keys, and they must be changed to `client-email:`,`client-id` and `private-key`, with dashes, in order to be recognised. 
```

Comment:
because 'keys' are what they are called in the context of a YAML file, I guess to distinguish them from the actual value. Perhaps it would be clearer if it said 'key names'